lib/travis/testing/scenario.rb

```diff
@@ -69,7 +69,7 @@ def default
       build :repository => enginex,
             :owner => enginex.owner,
             :number => 1,
-            :state  => 'failes',
+            :state  => 'fails',
```

Comment:
I can't imagine that `failes` is correct, but I could be wrong.
If it's intentional, it deserves a comment.